source/aggregation-pipeline-builder.txt

```diff
@@ -370,6 +374,35 @@ To create a view from your pipeline results:
 
 .. include:: /includes/fact-create-view-from-pipeline.rst
 
+.. _refresh-document-counts:
+
+Refresh Document Counts
+-----------------------
+
+When you delete or add a document, the :guilabel:`count results` value 
+on the :guilabel:`Aggregations` tab does not automatically update to 
+reflect the new document count.
```

Comment:
Sorry, I wasn't as clear as I should've been in my previous comment. This copy looks good, but I still think we want to retain the lead-in you had to these steps.

```suggestion
reflect the new document count.

To update the :guilabel:`count results`:
```

Reply:
Ok I think I am understanding you now, rewrite the first sentence and keep the lead in